client/request.go

```diff
@@ -178,7 +178,7 @@ func (cli *Client) doRequest(ctx context.Context, req *http.Request) (serverResp
 		// this is localised - for example in French the error would be
 		// `open //./pipe/docker_engine: Le fichier spécifié est introuvable.`
 		if strings.Contains(err.Error(), `open //./pipe/docker_engine`) {
-			err = errors.New(err.Error() + " In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.")
+			err = errors.New("In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running. " + err.Error())
```

Comment:
Not new (original code already did so), but this should  use `errors.Wrap` so that we preserve the original error;

```suggestion
			err = errors.Wrap(err, "in the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running. ")
```

Reply:
I've done the change to wrap the error and used @vikramhh's sample below to check for if the user is privileged so as to direct the error message